Detectors/FIT/common/simulation/include/FITSimulation/HitType.h

```diff
@@ -0,0 +1,30 @@
+#ifndef ALICEO2_FIT_HITTYPE_H_
+#define ALICEO2_FIT_HITTYPE_H_
+
+#include "SimulationDataFormat/BaseHits.h"
+#include "SimulationDataFormat/Stack.h"
+#include "CommonUtils/ShmAllocator.h"
+
+namespace o2
+{
+namespace fit
+{
+class HitType : public o2::BasicXYZEHit<float>
+{
```

Comment:
Above the V0 derives its own Hit class directly from the BasicXYZEHit. If this class is for T0 only, why is it in the common dir.?